packages/neuron-ui/src/services/remote/remoteApiWrapper.ts

```diff
@@ -80,6 +80,7 @@ type Action =
   | 'generate-dao-deposit-all-tx'
   | 'start-withdraw-from-dao'
   | 'withdraw-from-dao'
+  | 'get_fee_rate_statis'
```

Comment:
`get_fee_rate_statis` should be `get_fee_rate_stats`

Reply:
Have been removed.